aq_zombie_finder/aq_zombie_finder.py

```diff
@@ -139,19 +140,20 @@ def aq_zombie_finder():
     user = args.user
     password = args.password
     openstack_ip = args.ip
-    output = args.output
+    output_location = args.output or "output"
```

Comment:
Here, it "output" the default string if `args.output = None`?

Reply:
I've moved the default to be set in the arguements to make it more readable.